src/workers/notifyNFTSalesWorker.ts

```diff
@@ -67,12 +68,33 @@ export default function newWorker(
           if (nftSale.buyer === project.mintAddress) {
             return;
           }
-
-          await notifyDiscordSale(discordClient, channel, nftSale);
+          if (channel) {
+            await notifyDiscordSale(discordClient, channel, nftSale)
+              .catch(err => catchError(err, "Discord"));
```

Comment:
I think if you remove this line or wrap this in a try catch, the test should pass.

Reply:
Done!